.github/workflows/release.yml

```
@@ -313,15 +313,12 @@ jobs:
       - name: Install dependencies and pyogrio wheel
         shell: bash
         run: |
-          if [ ${{ matrix.python-version }} != "3.14" ]; then
-            uv pip install -r ci/requirements-wheel-test.txt
-          else
-            uv pip install pytest numpy certifi packaging
-          fi
+          uv pip install -r ci/requirements-wheel-test.txt
           uv pip install --no-cache --pre --no-index --find-links wheelhouse pyogrio
-          if [ ${{ matrix.python-version }} != "3.14" ]; then
-            uv pip install --no-deps geopandas
+          if [ ${{ matrix.python-version }} == "3.14t" ]; then
+            uv pip install --pre --extra-index-url https://pypi.anaconda.org/scientific-python-nightly-wheels/simple pandas
```

```suggestion
            uv pip install --pre --upgrade --extra-index-url https://pypi.anaconda.org/scientific-python-nightly-wheels/simple pandas
```

This might be needed since pandas is already installed, to ensure the nightly version actually gets installed